allimpl.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+
+	openapi "github.com/acceleratedlife/backend/go"
+	bolt "go.etcd.io/bbolt"
+)
+
+func UserByIdTx(tx *bolt.Tx, userId openapi.RequestUser) (user *bolt.Bucket, err error) {
+	users := tx.Bucket([]byte("users"))
```

Comment:
main.go includes constants which you need to use instead of common bucketNames
here 	KeyUsers        = "users"